docs/data/date-pickers/date-range-picker/ResponsiveDateRangePickers.tsx

```diff
@@ -30,6 +31,11 @@ export default function ResponsiveDateRangePickers() {
         <DemoItem label="Static variant">
           <StaticDateRangePicker
             defaultValue={[dayjs('2022-04-07'), dayjs('2022-04-10')]}
+            sx={{
```

Comment:
Fixes https://github.com/mui/mui-x/pull/7378#discussion_r1065670519

It's this example who made me realize we were forwarding the `className` and `sx` to the view on the static pickers instead of the layout root.